src/analysis/seminorm.lean

```diff
@@ -78,23 +205,30 @@ instance : inhabited (seminorm 𝕜 E) := ⟨0⟩
 
 variables (p : seminorm 𝕜 E) (c : 𝕜) (x y : E) (r : ℝ)
 
+protected lemma zero : p 0 = 0 := p.map_zero'
```

Comment:
It's a bit unfortunate that one version is called `zero` and the other one is `map_zero`, but I have no idea which to keep.